Jenkinsfile

```diff
@@ -15,6 +15,11 @@
 // limitations under the License.
 // ------------------------------------------------------------------------------
 
+// Discard old builds after 31 days
+properties([[$class: 'BuildDiscarderProperty', strategy:
+        [$class: 'LogRotator', artifactDaysToKeepStr: '',
+        artifactNumToKeepStr: '', daysToKeepStr: '31', numToKeepStr: '']]]);
+
```

Comment:
artifactDaysToKeepStr results in a different effect: "if not empty, artifacts from builds older than this number of days will be deleted, but the logs, history, reports, etc for the build will be kept"

We'll be triggering this job out of band, so I left out scheduled runs here.